bin/i18n/i18n_script_utils.rb

```diff
@@ -82,7 +82,7 @@ def self.run_bash_script(location)
     I18nScriptUtils.run_standalone_script("bash #{location}")
   end
 
-  private_class_method def self.report_malformed_restoration(key, translation, file_name)
+  def self.report_malformed_restoration(key, translation, file_name)
```

Comment:
Why change this from a private method to a public one?

Reply:
Laziness, mostly. :D It was throwing an error, and private class methods in rails are weird enough that I decided not to bother investigating